docs/framework/migration-guide/application-compatibility.md

```diff
@@ -1,142 +1,110 @@
 ---
 title: "Application Compatibility in the .NET Framework | Microsoft Docs"
 ms.custom: ""
-ms.date: "03/30/2017"
+ms.date: "05/01/2017"
 ms.prod: ".net-framework"
 ms.reviewer: ""
 ms.suite: ""
-ms.technology: 
+ms.technology:
```

Comment:
That sounds like a good idea, do you want me to fill it in with "ms.technology: app-compat" in the meantime?